### ISSUE TITLE
Why are negative quantities of a resource allowed?

### ISSUE THREAD

**Issue by sbocconi** [reflow-project/weloop#110]
I can keep on transferring a resource that the provider does not have anymore, as in the screenshot below where the quantity is -1 kilo.

![Screenshot 2021-08-11 at 16 45 21](https://user-images.githubusercontent.com/3026280/129050934-5167237b-4fe3-4e06-9805-99cf769b69df.png)

**Comment by pral2a** [comment]
Yes, this is a backend related issue. We post it here https://github.com/dyne/zenpub/issues/59
